static/tests/frontend/specs/_utils.js

```diff
@@ -64,6 +64,12 @@ ep_cursortrace_test_helper.utils = {
     return { top: top, left: left };
   },
 
+  getDistanceBetweenCaretIndicatorAndBeginningOfLine: function(lineNumber) {
+    var utils = ep_cursortrace_test_helper.utils;
```

Comment:
isso não é o `this`?

Reply:
Sim, é o `this`. Mas o valor de `this` pode mudar, dependendo do contexto. Na grande maioria das vezes é o próprio `utils`, mas nem sempre -- já tive muito problema nos testes por dar erro por causa disso, daí parei de usar o `this` por padrão.